packages/twenty-front/src/modules/client-config/utils/clientConfigUtils.ts

```diff
@@ -0,0 +1,38 @@
+import { ClientConfig } from '~/generated/graphql';
```

Comment:
we keep importing this from graphql as we moved it to a controller. I expect this to disappear from generated graphql files as we move further